lib/index.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
I'm not sure whether it's good enough to prepend this in the browserified version. This is also shipped and executed as-is in node.